### PR TITLE
Create home screen

### PR DESCRIPTION
**Pull request by amyqiu**
Issue #92 

Added home screen framework, currently has link to FAQs

Home screen (updated): 
https://user-images.githubusercontent.com/15674572/36921447-68d81cea-1e32-11e8-81c4-a11bf31e4ce7.png

Clicking on FAQ brings you to:
https://user-images.githubusercontent.com/15674572/36907483-c6671356-1e06-11e8-9aeb-7b4791b24f5f.png

### CODE REVIEW

File: app/components/home/Home.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	}
+}
+
+Home.propTypes = {
+	navigation: PropTypes.object.isRequired,
+	navigate: PropTypes.func,
+};
+
+Home.defaultProps = {
+	navigate:() => {}
```
Comment (theRoughCode) [inline app/components/home/Home.js]: missing space here

**Comment by amyqiu** [comment]
Fixed spacing and added screenshots

**Comment by theRoughCode** [comment]
@amyqiu Could you change the logo to a home logo?

https://material.io/icons/#ic_home

**Comment by amyqiu** [comment]
Updated home icon!

**Pull request merged**